buildchow/templates/core-jobs.yaml

```diff
@@ -5,9 +5,11 @@
       master
     devstudio-stream:
       master
-    target-platform-version: 4.50.0.Beta1-SNAPSHOT
-    target-platform-version-maximum: 4.50.0.Beta1-SNAPSHOT
-    disabled: true
+    target-platform-version: 4.50.0.CR1-SNAPSHOT
+    target-platform-version-maximum: 4.50.0.CR1-SNAPSHOT
+    blockBuildWhenDownstreamBuilding: false
```

Comment:
Some jobs are linked (aggregates, JBDS installer, tests). Most are not (jbosstools-* component jobs). So it helps the linked jobs and doesn't do anything for the unlinked ones. 

Your user story of "changes in base blocks rebuild of openshift" is impossible given the jobs are currently not linked. If they were, templating would be much harder / less standardized as we'd have to codify all the linkages.